### PR TITLE
L8

### PR DESCRIPTION
**Pull request by mstaack**

### CODE REVIEW

**Review by eisfeuer — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     "php": "^7.4",
     "ext-json": "*",
     "astrotomic/laravel-guzzle": "^2.1",
-    "illuminate/cache": "^6.0 || ^7.0",
-    "illuminate/support": "^6.0 || ^7.0"
+    "illuminate/cache": "^6.0 || ^7.0 || ^8.0",
+    "illuminate/support": "^6.0 || ^7.0 || ^8.0"
```
Comment (Gummibeer) [inline composer.json:25]: `orchestra/testbench` fehlt noch

**Review by Gummibeer — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (mstaack) [reply composer.json:25]: done

Comment (Gummibeer) [reply composer.json:25]: https://github.com/elbgoods/laravel-trashmail-rule/pull/15/commits/bd97f4132ed8f143fc57ddcf93886eda3e9fb321

**Review by Gummibeer — APPROVED** [review]

**Comment by Gummibeer** [comment]
https://github.com/elbgoods/laravel-trashmail-rule/releases/tag/v0.8.0

**Pull request merged**